### ISSUE TITLE
Add Artificial Animal Jaws

### ISSUE THREAD

**Issue by kaptain-kavern** [kaptain-kavern/ADogSaid#4]
Also the Animal Jaws seem to be missing completely from the mod (which is kinda bad bc w/o a jaw they starve to death ;P)

**Comment by kaptain-kavern** [comment]
Will be in next update
